local-container-endpoints/config/config.go

```diff
@@ -39,6 +43,10 @@ const (
 	DefaultTaskARN       = "arn:aws:ecs:us-west-2:111111111111:task/ecs-local-cluster/37e873f6-37b4-42a7-af47-eac7275c6152"
 	DefaultTDFamily      = "esc-local-task-definition"
 	DefaultTDRevision    = "1"
+
+	// Custom endpoint related
+	DefaultIAMCustomEndpoint = ""
+	DefaultSTSCustomEndpoint = ""
```

Comment:
These aren't actually the default endpoints... see my suggested re-write for `NewCredentialService`- these can be removed.

Reply:
Done.